src/classes/CON_PrimaryAffls_TEST.cls

```diff
@@ -787,7 +823,7 @@ private with sharing class CON_PrimaryAffls_TEST {
         insert businessAccount1;
 
         List<Contact> contactsToBeInserted = new List<Contact>();
-        for (integer i = 0; i < 5; i++) {
+        for (integer i = 0; i < 100; i++) {
```

Comment:
Now that's a bulk test! We may be able to pare this down a bit in the future and move anything over 5 records it to our unmanaged tests, but this makes me feel a touch more comfortable for what we're doing right now. On a different note, consider instantiating a static integer for the number of contacts to test. That would let us change it more rapidly.